apps/site/src/app/(main)/(home)/sections/FAQ/FAQAccordionDesktop.tsx

```diff
@@ -0,0 +1,140 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+
+import ListItemButton from "./components/ListItemButton";
+import TriangleIcon from "./components/TriangleIcon";
+import { FAQAccordion, FAQ } from "./FAQ";
+
+import SpeechDesktop from "./assets/speech-desktop.svg";
+
+const FAQAccordionDesktop: React.FC<FAQAccordion> = ({ faq }) => {
+	const faqGroup1 = faq.slice(0, 8);
+	const faqGroup2 = faq.slice(8);
+
+	const [page1Selected, setPage1Selected] = useState<boolean>(true);
+
+	const [focusedQuestion, setFocusedQuestion] = useState<null | FAQ>(null);
+
+	return (
+		<div className="flex flex-1 justify-center lg:h-auto mt-16 lg:mt-8 mb-[-250px] lg:mb-0">
+			{/* Desktop View */}
+			<Image
+				src={SpeechDesktop}
+				alt="Dialogue box background"
+				className="hidden lg:block h-fit  lg:max-w-[630px] lg:h-[310px] xl:max-w-[775px] xl:h-[350px] duration-300"
+			/>
+			<div className="hidden lg:block absolute lg:w-[555px] xl:w-[685px] lg:text-[.85rem] xl:text-[1rem] mt-3">
+				{/* Page 1 focused */}
+				<div
+					className={`absolute w-full duration-300 ${
+						page1Selected && !focusedQuestion
+							? "opacity-100 z-50"
+							: "opacity-0 pointer-events-none"
+					}`}
+				>
+					{faqGroup1.map((F) => (
+						<ListItemButton
+							key={F._key}
+							onClick={() => setFocusedQuestion(F)}
+							text={F.question}
+							inverted={false}
+							className="mb-0 py-[2px]"
+						/>
+					))}
+					<div className="flex justify-end w-full ms-3">
+						<button
+							type="button"
+							onClick={() => {
+								setPage1Selected(false);
+							}}
+							className={`flex items-center opacity-100 hover:bg-white hover:text-black p-[1px] px-[3px] duration-200 group`}
+						>
+							Page 1/2
+							<TriangleIcon className="ms-2 opacity-0 lg:w-4 lg:h-4 xl:w-5 xl:h-5" />
+							<TriangleIcon
+								className={`ms-2 absolute opacity-0 group-hover:opacity-100 duration-200 right-[-7px] lg:w-4 lg:h-4 xl:w-5 xl:h-5`}
+								dark
+							/>
+							<TriangleIcon
+								className={`ms-2 absolute opacity-100 group-hover:opacity-0 duration-200 right-[-7px] lg:w-4 lg:h-4 xl:w-5 xl:h-5`}
```

Comment:
Suggestion [repeating]: rather than using `opacity` to show and hide stuff, use `hidden` instead